ios/MaasNearbyTransit/Sources/MaasNearbyTransit/Components/NearbyTransitScheduleBadgesScrollView.swift

```diff
@@ -0,0 +1,214 @@
+import SwiftUI
+import Swappable
+
+private struct GradientScrollDataPreferenceKey: PreferenceKey {
+    typealias Value = ScrollViewData
+
+    static var defaultValue: ScrollViewData = .init(size: .zero, contentOffset: 0, contentSize: .zero)
+
+    static func reduce(value: inout ScrollViewData, nextValue: () -> ScrollViewData) {
+        value = nextValue()
+    }
+}
+
+private struct GradientBoundsScrollView<Content>: View where Content: View {
```

Comment:
In the future please try to stick to one topic / change per PR - these UI components are unrelated to the package structure refactor.

Reply:
Sure, I'll just remove one folder out of repo at the moment.

Reply:
It's fine, don't worry about it for this PR, as some of the nearby transit stuff was already in main so there will be a diff either way 🙂 Just wanted to share the feedback for the future.